docker-hub/vulnerability-scanning.md

```diff
@@ -10,15 +10,34 @@ title: Hub Vulnerability Scanning
   target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_scan"
 %}
 
-Docker Hub Vulnerability Scanning enables you to automatically scan Docker images for vulnerabilities using Snyk. This uses the same technology as the [docker scan](../engine/scan/index.md) command.
+Docker Hub Vulnerability Scanning enables you to automatically scan Docker
+images for vulnerabilities using Snyk. This uses the same technology as the
+[docker scan](../engine/scan/index.md) command.
+
+> When you enable Hub Vulnerability Scanning, you can also see whether your
+> images are affected by the Log4j 2 CVE-2021-44228. For more information, see
```

Comment:
I think we need to settle on a way to reference this vulnerability. On Hub, we say things like:

`affected by CVE-2021-44228 (a.k.a. Log4Shell)`

...which I think is a good mix of the formal and informal names. "Log4j 2" just specifies a version range of the library.

For the docs, I'd lean toward things like:

```suggestion
> images are affected by Log4Shell (CVE-2021-44228). For more information, see
```